pages/common/fping.md

```diff
@@ -0,0 +1,15 @@
+# fping
+
+> A more powerful ping which can ping multiple hosts.
+
+- List alive hosts within a given subnet using a netmask:
+
+`fping -a -g 192.168.1.0/24`
```

Comment:
@sbrl the command does not generate the list of alive hosts: it generates a subnet, tests all hosts within it for aliveness, and returns the list of alive ones. I would rather go with something closer to @agnivade's proposal: `List alive hosts within a subnet generated from a netmask`.